### PR TITLE
Added a generator to use Phoenix's new dir structure.

### PR DESCRIPTION
**Pull request by yuchunc**
This PR need to wait for [plug_rest PR#4](https://github.com/christopheradams/plug_rest/pull/4) to be published.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -21,7 +21,7 @@ defmodule PhoenixRest.Mixfile do
   end
 
   defp deps do
-    [{:phoenix, "~> 1.2"},
+    [{:phoenix, "~> 1.2-rc"},
```
Comment (christopheradams) [inline mix.exs]: This should be 1.3.0-rc

File: mix.exs [hunk 2]
```diff
@@ -21,7 +21,7 @@ defmodule PhoenixRest.Mixfile do
   end
 
   defp deps do
-    [{:phoenix, "~> 1.2"},
+    [{:phoenix, "~> 1.3.0-rc"},
      {:plug_rest, "~> 0.11"},
```
Comment (christopheradams) [inline mix.exs]: Now you can upgrade to plug_rest 0.12

Comment (yuchunc) [reply mix.exs]: :+1: I'll update this, also will find some time to test.

Comment (christopheradams) [reply mix.exs]: Looks ok now. Let me know when it's ready to merge.

Comment (yuchunc) [reply mix.exs]: Hi @christopheradams, sorry had a frantic week.
I tested it on my pet project, should be good!

Thanks for merging~ = )

**Pull request merged**